src/BotInterface/Wallet.hs

```diff
@@ -0,0 +1,96 @@
+module BotInterface.Wallet (
+  BpiWallet,
+  usingEnv,
+  addSomeWallet,
+  mkMainnetAddress,
+  cardanoMainnetAddress,
+) where
+
+import BotInterface.Setup qualified as Setup
+import BotInterface.Types
```

Comment:
This should be a qualified import or explicit import list

Reply:
addressed